.github/workflows/cluster_endtoend.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        name: [11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25]
+        name: [11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24]
```

Comment:
It looks like 25 might have been removed by accident? There is a test below that requires it.

Reply:
Please check the[ 3rd answer](https://github.com/vitessio/vitess/pull/5866#discussion_r388415704).